### ISSUE TITLE
Dependency conflicts 

### ISSUE THREAD

**Issue by UtkarshBhavsar** [Impetus/kundera#745]
Dependency conflicts in hadoop commons for habse. Both 2.5.1 and 2.6.0 is used. 

<pre>
Dependency convergence error for org.apache.hadoop:hadoop-common:2.5.1 paths to dependency are:
+-com.utkarsh.hbasedemo:hbase:1.0.7-SNAPSHOT
  +-com.impetus.kundera.client:kundera-hbase-v2:2.17
    +-org.apache.hbase:hbase-client:1.0.0
      +-org.apache.hadoop:hadoop-common:2.5.1
and
+-com.utkarsh.hbasedemo:hbase:1.0.7-SNAPSHOT
  +-com.impetus.kundera.client:kundera-hbase-v2:2.17
    +-org.apache.hbase:hbase-server:1.0.0
      +-org.apache.hbase:hbase-prefix-tree:1.0.0
        +-org.apache.hbase:hbase-common:1.0.0
          +-org.apache.hadoop:hadoop-common:2.5.1
and
+-com.utkarsh.hbasedemo:hbase:1.0.7-SNAPSHOT
  +-com.impetus.kundera.client:kundera-hbase-v2:2.17
    +-org.apache.hbase:hbase-server:1.0.0
      +-org.apache.hbase:hbase-prefix-tree:1.0.0
        +-org.apache.hadoop:hadoop-common:2.5.1
and
+-com.utkarsh.hbasedemo:hbase:1.0.7-SNAPSHOT
  +-com.impetus.kundera.client:kundera-hbase-v2:2.17
    +-org.apache.hbase:hbase-server:1.0.0
      +-org.apache.hbase:hbase-hadoop2-compat:1.0.0
        +-org.apache.hadoop:hadoop-common:2.5.1
and
+-com.utkarsh.hbasedemo:hbase:1.0.7-SNAPSHOT
  +-com.impetus.kundera.client:kundera-hbase-v2:2.17
    +-org.apache.hbase:hbase-server:1.0.0
      +-org.apache.hadoop:hadoop-common:2.5.1
and
+-com.utkarsh.hbasedemo:hbase:1.0.7-SNAPSHOT
  +-com.impetus.kundera.client:kundera-hbase-v2:2.17
    +-org.apache.hbase:hbase-server:1.0.0
      +-org.apache.hadoop:hadoop-client:2.5.1
        +-org.apache.hadoop:hadoop-common:2.5.1
and
+-com.utkarsh.hbasedemo:hbase:1.0.7-SNAPSHOT
  +-com.impetus.kundera.client:kundera-hbase-v2:2.17
    +-org.apache.hbase:hbase-common:1.0.0
      +-org.apache.hadoop:hadoop-common:2.5.1
and
+-com.utkarsh.hbasedemo:hbase:1.0.7-SNAPSHOT
  +-com.impetus.kundera.client:kundera-hbase-v2:2.17
    +-org.apache.hadoop:hadoop-common:2.6.0
 </pre>

**Comment by pragalbhgarg** [comment]
@UtkarshBhavsar 

Have you explicitly added any dependency for **`hadoop-commons`** in your `pom.xml` ? If possible, please show us your `pom.xml`. 

-PG

**Comment by UtkarshBhavsar** [comment]
@pragalbhgarg 
I haven't added the dependency explicitly.

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>com.utkarsh.hbasedemo</groupId>
    <artifactId>utkarsh</artifactId>
    <version>1.0-SNAPSHOT</version>
    <repositories>
        <repository>
            <id>sonatype-nexus</id>
            <url>https://oss.sonatype.org/content/repositories/releases</url>
            <releases>
                <enabled>true</enabled>
            </releases>
            <snapshots>
                <enabled>false</enabled>
            </snapshots>
        </repository>
    </repositories>
    <dependencies>
        <dependency>
            <groupId>com.impetus.kundera.client</groupId>
            <artifactId>kundera-hbase-v2</artifactId>
            <version>2.17</version>
        </dependency>
    </dependencies>
    <build>
        <plugins>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-install-plugin</artifactId>
                <version>2.5.1</version>
                <configuration>
                    <skip>true</skip>
                </configuration>
            </plugin>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-deploy-plugin</artifactId>
                <version>2.8.1</version>
                <configuration>
                    <skip>true</skip>
                </configuration>
            </plugin>
        </plugins>
    </build>
</project>
```

**Comment by pragalbhgarg** [comment]
@UtkarshBhavsar 

Please try excluding the `hadoop-common` dependency from `kundera-hbase-v2` by doing the following in your `pom.xml`

```
        <dependency>
            <groupId>com.impetus.kundera.client</groupId>
            <artifactId>kundera-hbase-v2</artifactId>
            <version>2.17</version>
            <exclusions>
                <exclusion>
                    <groupId>org.apache.hadoop</groupId>
                    <artifactId>hadoop-common</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
```

-PG

**Comment by chhavigangwal** [comment]
Fixed with Kundera 3.0 release.
